cdk/lib/admin-console.ts

```diff
@@ -269,16 +244,19 @@ export class AdminConsole extends GuStack {
     const archivedBannerDesignsDynamoTable = this.buildArchivedBannerDesignsTable();
     const permissionsTable = this.buildPermissionsTable();
 
-    const channelTestsDynamoPolicies =
-      this.buildChannelTestsDynamoPolicies(channelTestsDynamoTable);
-    const campaignsDynamoPolicies = this.buildDynamoPolicies(campaignsDynamoTable);
-    const archivedTestsDynamoPolicies = this.buildDynamoPolicies(archivedTestsDynamoTable);
-    const channelTestsAuditDynamoPolicies = this.buildDynamoPolicies(channelTestsAuditDynamoTable);
-    const bannerDesignsDynamoPolicies = this.buildDynamoPolicies(bannerDesignsDynamoTable);
-    const archivedBannerDesignsDynamoPolicies = this.buildDynamoPolicies(
+    const dynamoPolicy = new DynamoPolicy(this, `DynamoPolicy`, [
+      channelTestsDynamoTable,
+      campaignsDynamoTable,
+      archivedTestsDynamoTable,
+      channelTestsAuditDynamoTable,
+      bannerDesignsDynamoTable,
       archivedBannerDesignsDynamoTable,
-    );
-    const permissionsDynamoPolicies = this.buildDynamoPolicies(permissionsTable);
+      permissionsTable,
+    ]);
+
+    const channelTestsIndexPolicy = new GuDynamoDBReadPolicy(this, `DynamoRead-${channelTestsDynamoTable.node.id}/index/campaignName-name-index`, {
+      tableName: `${channelTestsDynamoTable.tableName}/index/campaignName-name-index`,
+    });
```

Comment:
we still have to separately create this policy for an index on the channel-tests table